### ISSUE TITLE
Hardcoded directory separators

### ISSUE THREAD

**Issue by timdodge** [four-o-four/omniverse#5]
Hi, I'm trying to run this on a linux box - it looks like the directory separators have been hardcoded windows-style.

My db and log files are named:

db\headers.db and log\omniverse.log

rather than being in their own subdirectories.

There's some info about this here: http://docs.python.org/distutils/setupscript.html

**Comment by four-o-four** [comment]
You are right, everything is very windows-centric. I'll address it in the next push.

**Comment by timdodge** [comment]
Great, thanks.

It's looking very promising.

The other problem I was having is that the server only binds to localhost, so I can't browse from another machine. So I stuck in this line, just before the socket_port one:

cherrypy.server.socket_host = '0.0.0.0'

**Comment by four-o-four** [comment]
Thanks for pointing that out. Added an issue for it as an enhancement.
